src/libsyntax/parse/lexer/tokentrees.rs

```diff
@@ -11,7 +11,7 @@
 use print::pprust::token_to_string;
 use parse::lexer::StringReader;
 use parse::{token, PResult};
-use tokenstream::{DelimSpan, TokenStream, TokenTree};
+use tokenstream::{DelimSpan, IsJoint::*, TokenStream, TokenTree};
```

Comment:
This is a neat trick, cc @nikomatsakis

Reply:
I was surprised but happy that it worked :smile: